tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php

```diff
@@ -1406,6 +1406,58 @@ class Foo {
     public function doFoo(iterable $bar, ?int $baz): ?array {}
 }',
             ],
+            'abstract annotation in function' => [
```

Comment:
Please add some tests that show the annotations are not removed when the class or the functions are not marked as final/abstract.